logic/theme-changer.js

```diff
@@ -0,0 +1,32 @@
+//* dark mode
+
+let currentTheme = localStorage.getItem('theme');
+if (currentTheme === 'light') {
+	document.documentElement.setAttribute('data-theme', 'light');
+	document.querySelectorAll('.theme-changer-icon').forEach(function (item) {
+		item.src = '/assets/svg/moon.svg';
+	});
+} else {
+	document.documentElement.setAttribute('data-theme', 'dark');
+	document.querySelectorAll('.theme-changer-icon').forEach(function (item) {
+		item.src = '/assets/svg/sun.svg';
+	});
```

Comment:
why are we not using getElementsByClassName[0] here when we know that there is only one icon to toggle theme . Using this eliminates forEach.

Reply:
Can do this as there will be many "theme-changer-icon" on multiple pages this crash.